core_lib/src/canvaspainter.cpp

```diff
@@ -552,6 +615,30 @@ void CanvasPainter::renderGrid(QPainter& painter)
     }
 }
 
+void CanvasPainter::renderOverlays(QPainter &painter)
+{
+    if (mOptions.bCenter)
+    {
+        painter.setWorldTransform(mViewTransform);
```

Comment:
If you need to call setWorldTransform in each condition, then you could probably move that into the paintOverlays method.

Reply:
You're right